modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/enricher/DynamicMetadataDeclarationEnricher.java

```diff
@@ -152,9 +175,10 @@ private void enrichWithDsql(OperationDeclaration declaration, MethodElement meth
                                                                                                                        .nativeOutputResolver(),
                                                                                                                    query
                                                                                                                        .entityResolver())));
+
       addQueryModelProperties(declaration, query);
       declareDynamicType(declaration.getOutput());
-      declareMetadataKeyId(declaration, new NullMetadataScopeAdapter());
+      declareMetadataKeyId(declaration, null);
```

Comment:
is this ok?

Reply:
I mean the “null”

Reply:
sadly, it is, we use null everywhere to mark a non-existing categoryName

Reply:
is this allowed in the model? when is the category name absent?